src/panels/lovelace/hui-root.ts

```diff
@@ -0,0 +1,583 @@
+      .open="${this.notificationsOpen}"
+      @open-changed="${this._handleNotificationsOpenChanged}"
+      .narrow="${this.narrow}"
+    ></hui-notification-drawer>
+    <ha-app-layout id="layout">
+      <app-header slot="header" effects="waterfall" fixed condenses>
+        ${
+          this._editMode
+            ? html`
+                <app-toolbar>
+                  <paper-icon-button
+                    icon="hass:close"
+                    @click="${this._editModeDisable}"
+                  ></paper-icon-button>
+                  <div main-title>
+                    ${this.localize("ui.panel.lovelace.editor.header")}
+                  </div>
+                </app-toolbar>
+              `
+            : html`
+                <app-toolbar>
+                  <ha-menu-button
+                    .narrow="${this.narrow}"
+                    .showMenu="${this.showMenu}"
+                  ></ha-menu-button>
+                  <div main-title>${this.config.title || "Home Assistant"}</div>
+                  <hui-notifications-button
+                    .hass="${this.hass}"
+                    .notificationsOpen="{{notificationsOpen}}"
+                    .notifications="${this._notifications}"
+                  ></hui-notifications-button>
+                  <ha-start-voice-button
+                    .hass="${this.hass}"
+                  ></ha-start-voice-button>
+                  <paper-menu-button
+                    no-animations
+                    horizontal-align="right"
+                    horizontal-offset="-5"
+                  >
+                    <paper-icon-button
+                      icon="hass:dots-vertical"
+                      slot="dropdown-trigger"
+                    ></paper-icon-button>
+                    <paper-listbox
+                      @iron-select="${this._deselect}"
+                      slot="dropdown-content"
+                    >
+                      ${
+                        this._yamlMode
+                          ? html`
+                              <paper-item @click="${this._handleRefresh}"
+                                >Refresh</paper-item
+                              >
+                            `
+                          : ""
+                      }
+                      <paper-item @click="${this._handleUnusedEntities}"
+                        >Unused entities</paper-item
+                      >
+                      <paper-item @click="${this._editModeEnable}"
+                        >${
+                          this.localize("ui.panel.lovelace.editor.configure_ui")
+                        }</paper-item
+                      >
+                      ${
+                        this._storageMode
+                          ? html`
+                              <paper-item
+                                @click="${this.lovelace!.enableFullEditMode}"
+                                >Raw config editor</paper-item
```

Comment:
I know this is just a convert but this, unused entities and Help should be localized yes?